istioctl/cmd/deprecated_cmd.go

```diff
@@ -44,12 +44,13 @@ import (
 	"istio.io/api/networking/v1alpha3"
 	"istio.io/pkg/log"
 
+	"istio.io/istio/galley/pkg/config/schema/collection"
```

Comment:
We should probably just delete this file / commands. They've been deprecated for more than a year now.

Reply:
@esnible, any objection to removing the deprecated CRUD commands?